template/cordova/lib/MSBuildTools.js

```diff
@@ -50,11 +50,11 @@ MSBuildTools.prototype.buildProject = function(projFile, buildType, buildarch, o
 
 // returns full path to msbuild tools required to build the project and tools version
 module.exports.findAvailableVersion = function () {
```

Comment:
We would also need to update [`findAllAvailableVersions` method](https://github.com/apache/cordova-windows/pull/133/files#diff-cb73739fe4756cb409aeb5941bbb8e47R64)